App/NatronApp_main.cpp

```diff
@@ -94,6 +94,14 @@ int main(int argc, char *argv[])
             freopen("CONOUT$", "w", stderr);
         }
     }
+#if QT_VERSION < QT_VERSION_CHECK(6, 5, 0)
+    // Set proper window title bar color (https://www.qt.io/blog/dark-mode-on-windows-11-with-qt-6.5)
```

Comment:
I'm assuming you got the registry code from
https://learn.microsoft.com/en-us/answers/questions/1161597/how-to-detect-windows-application-dark-mode

Should we add a reference to that URL as well?

Reply:
Sure, added the url.